config/sidekiq.yml

```diff
@@ -61,6 +61,10 @@
     cron: "0 4 * * *"
     description: "For Publishers who have stale uphold codes or access parameters, clean out uphold codes or access parameter."
     queue: scheduler
+  Sync::Bitflyer::UpdateMissingDepositsJob:
```

Comment:
Is this a 1 time fix or do we need to run it continuously?

Reply:
Might be a long time run continuously as we don't know the up time of the provider.

Reply:
I should clarify: what's the source of the missing data? If we plug that hold then we can just run this once.

Reply:
The issue is that even if we fix the data now, it's possible that the deposit_id will be missing in the future, i.e. if Bitflyer's API goes down, then it's best to automatically re-fetch the deposit ID eventually.